backend/spacewalk-backend.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Thu Aug 19 17:06:27 CEST 2021 - ebischoff@suse.com
+
+- fix typo "verfication" instead of "verification"
```

Comment:
You don't need to add the header now, only you changelog message at the top of the file.

The headers will be added by Julio when tagging for the release.
```suggestion
- fix typo "verfication" instead of "verification"
```

Reply:
Done, thanks